cernopendata/modules/fixtures/data/records/cms-derived-nanoaod-outreach-higgstautau.json

```diff
@@ -0,0 +1,2756 @@
+                "type": "float[nJet]",
+                "variable": "Jet_pt"
+            },
+            {
+                "description": "Pseudorapidity of the jets",
+                "type": "float[nJet]",
+                "variable": "Jet_eta"
+            },
+            {
+                "description": "Azimuth of the jets",
+                "type": "float[nJet]",
+                "variable": "Jet_phi"
+            },
+            {
+                "description": "Mass of the jets",
+                "type": "float[nJet]",
+                "variable": "Jet_mass"
+            },
+            {
+                "description": "Jet pile-up identification flag",
+                "type": "bool[nJet]",
+                "variable": "Jet_puId"
+            },
+            {
+                "description": "b-Jet identification score",
+                "type": "float[nJet]",
+                "variable": "Jet_btag"
+            },
+            {
+                "description": "Missing transverse energy",
+                "type": "float",
+                "variable": "MET_pt"
+            },
+            {
+                "description": "Azimuth of the missing transverse energy",
+                "type": "float",
+                "variable": "MET_phi"
+            },
+            {
+                "description": "Number of stored generator particles",
+                "type": "unsigned int",
+                "variable": "nGenPart"
+            },
+            {
+                "description": "Transverse momentum of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_pt"
+            },
+            {
+                "description": "Pseudorapidity of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_eta"
+            },
+            {
+                "description": "Azimuth of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_phi"
+            },
+            {
+                "description": "Mass of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_mass"
+            },
+            {
+                "description": "Particle identification number of the generator particles",
+                "type": "int[nGenPart]",
+                "variable": "GenPart_pdgId"
+            }
+        ],
+        "date_published": "2019",
+        "distribution": {
+            "formats": [
+                "root"
+            ],
+            "number_events": 476963,
+            "number_files": 1,
+            "size": 196517149
+        },
+        "experiment": "CMS",
+        "files": [
+            {
+                "checksum": "adler32:2b025925",
+                "size": 196517149,
+                "uri": "root://eospublic.cern.ch//eos/opendata/cms/derived-data/AOD2NanoAODOutreachTool/GluGluToHToTauTau.root"
```

Comment:
Currently we have:

```console
$ eos ls /eos/opendata/cms/derived-data/AOD2NanoAODOutreachTool/
Run2012BC_DoubleMuParked_Muons.root
```

After the PR, we would have more files in the same directory:

```
DYJetsToLL.root
GluGluToHToTauTau.root
Run2012B_TauPlusX.root
Run2012BC_DoubleMuParked_Muons.root
Run2012C_TauPlusX.root
TTbar.root
VBF_HToTauTau.root
W1JetsToLNu.root
W2JetsToLNu.root
W3JetsToLNu.root
```

Are the new data also derived using the same `AOD2NanoAODOutreachTool`?  If so, how would people know how they can generate the former and how the latter data? (E.g. parameters to use, commands to call, etc.)  If the same tool can be used to produce many different derived datasets for different purposes, then it would make sense invent some scalable storage schema, e.g. to have subdirectories there for each use case or some such?

Reply:
This is indeed the same tool, for the previous files some parts were commented in the code to produce only muons, here instead, the full output is kept.

Reply:
OK, I'll move them to the same place, but we should think how to capture the tool parametrisation leading to the various outputs. Could be simply via tags and using something like:

- `AOD2NanoAODOutreachTool/v1.0/Run2012BC_DoubleMuParked_Muons.root` 
- `AOD2NanoAODOutreachTool/v1.1/Run2012C_TauPlusX.root`
 
and the information about "provenance" will be found within the given tool's tag.

Or, if the same version of the tool is used, one could attach "provenance" configuration files and/or shell scripts next to each file:

- `AOD2NanoAODOutreachTool/Run2012C_TauPlusX.root` 
- `AOD2NanoAODOutreachTool/Run2012C_TauPlusX_generation/run.sh` 
- `AOD2NanoAODOutreachTool/Run2012C_TauPlusX_generation/config.h` 


that the given version of the tool uses. We can muse later about these...